.changeset/tender-penguins-carry.md

```diff
@@ -0,0 +1,13 @@
+---
+'@eventstore-ui/layout': minor
+---
+
+Nw component `es-sized-panel` has been added that can be placed in any layout area, with a custom start and end position. It will set the size of the corresponding area to it's own size, much like how `es-sidebar` works.
```

Comment:
```suggestion
New component `es-sized-panel` has been added that can be placed in any layout area, with a custom start and end position. It will set the size of the corresponding area to it's own size, much like how `es-sidebar` works.
```